docs/topics/mta-review-reports.adoc

```diff
@@ -46,11 +46,16 @@ The following table lists all of the reports and pages that can be accessed from
 
 Note that if an application shares archives with other analyzed applications, you will see a breakdown of how many story points are from shared archives and how many are unique to this application.
 
-.Shared archives
-image::3-2-shared-archives.png[Shared archives]
+// .Shared archives
+// image::3-2-shared-archives.png[Shared archives]

```

Comment:
MTA does not produce any Issues for the JEE example application.
@abrugaro Is there a test application that I can run that produces issues, shared archives, etc?